src/d3a/setup/strategy_tests/external_ess_offers.py

```diff
@@ -33,7 +33,7 @@ def get_setup(config):
                 'House 1',
                 [
                     Area('load', strategy=LoadHoursStrategy(
-                        hrs_of_day=list(range(24)), hrs_per_day=24, avg_power_W=4000,
+                        hrs_of_day=list(range(2, 24)), hrs_per_day=20, avg_power_W=4000,
```

Comment:
I guess this it to account for the integrationtest in d3a-api-client?

Reply:
Exactly, as I only want it to be tested with external_ess_agent's offer